src/vs/editor/contrib/referenceSearch/referencesWidget.ts

```diff
@@ -694,32 +638,31 @@ export class ReferenceWidget extends PeekViewWidget {
 				accessibilityProvider: new AriaProvider()
 			};
 
-			// listen on selection and focus
-			this._disposables.push(controller.onDidFocus((element) => {
-				if (element instanceof OneReference) {
-					this._revealReference(element);
-					this._onDidSelectReference.fire({ element, kind: 'show', source: 'tree' });
-				}
-			}));
+			var options: tree.ITreeOptions = {
+				twistiePixels: 20,
+				ariaLabel: nls.localize('treeAriaLabel', "References"),
+				keyboardSupport: false
+			};
 
-			this._disposables.push(controller.onDidSelect((element: any) => {
-				if (element instanceof OneReference) {
-					this._onDidSelectReference.fire({ element, kind: 'goto', source: 'tree' });
-				}
-			}));
-			this._disposables.push(controller.onDidOpenToSide((element: any) => {
+			this._tree = this._instantiationService.createInstance(WorkbenchTree, div.getHTMLElement(), config, options);
```

Comment:
Somewhere the 'cmd+click to open to side' feature is being lost here

Reply:
@jrieken fixed via https://github.com/Microsoft/vscode/pull/41275/commits/64f9e8a98b406d3d04f3934791b98b04d7845719